doc/sphinx/src/models.rst

```diff
@@ -751,6 +751,107 @@ is :math:`S_0`, and ``expconsts`` is a pointer to the constant array of length
 :math:`d_2` to :math:`d_{40}`. Expansion coefficients not used should be set to
 0.0.
 
+Mie-Gruneisen linear :math:`U_s`-:math:`u_p` EOS
+````````````````````````````````````````````````
+
+One of the most commonly-used EOS is the linear :math:`U_s`-:math:`u_p` version of the Mie-Gruneisen EOS. This EOS
+uses the Hugoniot as the reference curve and is extensively used in shock physics.
+This version implements the exact thermodynamic tempearture on the Hugoniot and also adds an entropy.
+
+The pressure follows the traditional Mie-Gruneisen form,
+
+.. math::
+
+    P(\rho, e) = P_H(\rho) + \rho\Gamma(\rho) \left(e - e_H(\rho) \right),
+
+Here the subscript :math:`H` is a reminder that the reference curve is a
+Hugoniot. :math:`\Gamma` is the Gruneisen parameter and the first approximation 
+is that :math:`\rho\Gamma(\rho)=\rho_0\Gamma(\rho_0)`
+which is the same assumption as in the Gruneisen EOS when :math:`b=0`.
+
+The above is an incomplete equation of state because it only relates the
+pressure to the density and energy, the minimum required in a solution to the
+Euler equations. To complete the EOS and determine the temperature and entropy, a constant
+heat capacity is assumed so that
+
+.. math::
+
+    T(\rho, e) = \frac{\left(e-e_H(\rho)\right)}{C_V} + T_H(\rho)
+
+Note the difference from the Gruneisen EOS described above. We still use a constant :math:`C_V`, 
+and it is usually taken at the reference temperature, but
+we now extrapolate from the temperature on the Hugoniot, :math: `T_H(\rho)`, and not 
+from the reference temperature, :math:`T_0`.
+
+With this consistent temperature we can derive an entropy in a similar way as for the Vinet EOS,
+.. math::
```

Comment:
```suggestion
With this consistent temperature we can derive an entropy in a similar way as for the Vinet EOS,

.. math::
```
This isn't displaying correctly either. I think it's just missing a carriage return.